elated-egrets/bangazon-internal-site
# refactored employe to training program relationship

## DESCRIPTION
added many to many field and deleted our manual intersection table per joes recommendations 

## How to pull down
Commands

```
git fetch --all
git checkout rm-refactor-employee-training-relationship
```

## Steps to test

1. Start virtual environment
1. Run the faker data generator
1. make sure database is still successfully generated with the correct tables and fields

## What you changed
deleted manual intersection table and added many to many field on employees

## What was added to read me
changed to reflect model changes

## Final Checklist
Before merging.
1. Has each file changed been reviewed?
2. Has the readme been updated to reflect changes made?
3. Has a teammate approved the PR?

Then Ship It!
